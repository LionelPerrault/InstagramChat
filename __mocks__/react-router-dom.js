export const Link = 'Link'
export const Switch = 'Switch'
export const Redirect = 'Redirect'
export const Route = 'Route'
export const NavLink = 'NavLink'
