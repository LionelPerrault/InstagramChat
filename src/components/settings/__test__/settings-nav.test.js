import React from 'react'
import { create } from 'react-test-renderer'
import { BrowserRouter as Router } from 'react-router-dom'
import SettingsNav from '../settings-nav'

describe('SettingsNav Component', () => {

  it('should match snapshot', () => {
    const tree = create(
      
      <SettingsNav
        url='/settings'
      />
      
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })

})
