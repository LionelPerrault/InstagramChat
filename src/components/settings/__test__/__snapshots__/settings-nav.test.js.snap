// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsNav Component should match snapshot 1`] = `
<div
  className="settings_nav_div"
>
  <ul>
    <li>
      <NavLink
        activeClassName="settings_nav_active"
        className="settings_nav"
        exact={true}
        to="/settings"
      >
        Profile settings
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="settings_nav_active"
        className="settings_nav"
        to="/settings/change-password"
      >
        Change password
      </NavLink>
    </li>
    <li>
      <NavLink
        activeClassName="settings_nav_active"
        className="settings_nav"
        to="/settings/deactivate"
      >
        Deactivate
      </NavLink>
    </li>
  </ul>
</div>
`;
